.github/run_kuttl_tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -ex
+
+# Install kuttl
+if [[ ! -f /usr/local/bin/kubectl-kuttl ]]; then
+    wget -O /usr/local/bin/kubectl-kuttl https://github.com/kudobuilder/kuttl/releases/download/v0.9.0/kubectl-kuttl_0.9.0_linux_x86_64
+    chmod +x /usr/local/bin/kubectl-kuttl
+fi
+
+
+OPERATOR=$(echo ${1} | cut -d '-' -f1)
+export ${OPERATOR^^}_IMG=$2/$1-index:$3
+
+echo $4
+echo $5
+export ${OPERATOR^^}_REPO=https://github.com/$4.git
+echo ${OPERATOR^^}_REPO
+export ${OPERATOR^^}_BRANCH=$5
+echo ${OPERATOR^^}_BRANCH
+
+cd $HOME
+rm -rf install_yamls
+git clone https://github.com/openstack-k8s-operators/install_yamls.git
+cd $HOME/install_yamls
+make crc_storage
+sleep 20
+make ${OPERATOR}_kuttl
```

Comment:
Does this make target run all tests? If something fail, it will retorn here?

Reply:
Yes, please see https://github.com/tripleo-ci/keystone-operator/actions/runs/3988080564/jobs/6954400156#step:7:2681